plugins/jobtype/jobtypes/teradataToHdfs/plugin.properties

```diff
@@ -0,0 +1,5 @@
+mr.listener.visualizer=false
```

Comment:
what is the difference between lib and libjars
plugins/jobtype/jobtypes/teradataToHdfs/lib/
plugins/jobtype/jobtypes/teradataToHdfs/libjars/ 
can you use just one, say lib ?